src/Components/UserSharedComponents/NavBar/NavBar.tsx

```diff
@@ -51,113 +62,157 @@ const [favoriteItemsCount, setFavoriteItemsCount] = useState(0)
+              }`}
+              to={"./explore"}
+            >
+              Explore
+            </Link>
+            <Link
+              className={`navLink ${
+                pathname === "/room-reviews" ? "activeLink" : ""
+              }`}
+              to={"./room-reviews"}
+            >
+              Reviews
+            </Link>
+            <IconButton aria-label={notificationsLabel(100)}>
+              <Badge
+              className="favoriteCount"
+                badgeContent={
+                  favoriteItemsCount === 0 ? "0" : favoriteItemsCount
+                }
+                color="primary"
               >
-                <MenuItem onClick={handleClose}>
-                  <Typography variant="caption" color="red" onClick={() => handelLogout()}>
-                    Log Out
+                <Favorite onClick={() => navigate("./favorite-rooms")} />
+              </Badge>
+            </IconButton>
+            {!localStorage.getItem("authToken") ? (
+              <>
+                <Button
+                  className="navBtn"
+                  variant="contained"
+                  onClick={() => navigate("./login")}
+                >
+                  Login Now
+                </Button>
+                <Button
+                  className="navBtnSign"
+                  variant="contained"
+                  onClick={() => navigate("./register")}
+                >
+                  Sign Up
+                </Button>
+              </>
+            ) : (
+              <>
+                <Box className="navAvatar">
+                  <Tooltip title="Open settings">
+                    <IconButton>
+                      <img
+                        className="nav-img"
+                        src={
+                          userData?.profileImage == null
+                            ? defaultImage
+                            : `http://res.cloudinary.com/dpa4yqvdv/image/upload/v1705784141/users/fod9w8mryr5c5raufxwc.jpg`
+                        }
+                        id="demo-positioned-button"
+                        aria-controls={
+                          open ? "demo-positioned-menu" : undefined
+                        }
+                        aria-haspopup="true"
+                        aria-expanded={open ? "true" : undefined}
+                        onClick={handleClick}
+                      />
+                    </IconButton>
+                  </Tooltip>
+                </Box>
+                <Menu
+                  id="demo-positioned-menu"
+                  aria-labelledby="demo-positioned-button"
+                  anchorEl={anchorEl}
+                  open={open}
+                  onClose={handleClose}
+                >
+                  <MenuItem onClick={handleClose}>
+                    <Typography
+                      variant="caption"
+                      color="red"
+                      onClick={() => handelLogout()}
```

Comment:
This function redirects the user to the login again, I think it would be better if we just remove the user's data and refresh to the landing page instead.